### PR TITLE
IBM K8S logs

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-553--logz-docs.netlify.app/shipping/log-sources/iks.html

### CODE REVIEW

File: _source/logzio_collections/_log-sources/iks.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+kubectl apply -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-rbac.yaml
+```
+
+Or, if you have a non-RBAC cluster, run this instead:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset.yaml
+```
+
+For container runtime Containerd, run:
```
Comment (mirii1994) [inline _source/logzio_collections/_log-sources/iks.md:71]:
```suggestion
Run:
```

For this integration there's only one option.

File: _source/logzio_collections/_log-sources/iks.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+
+## Disabling systemd input
+
+To suppress Fluentd system messages, set the environment variable `FLUENTD_SYSTEMD_CONF` to `disable` in your Kubernetes environment.
+
+### Disable Prometheus input plugins
+
+By default, the latest images launch `prometheus` plugins to monitor Fluentd.
+If you'd like to disable the Prometheus input plugin, set the environment variable `FLUENTD_PROMETHEUS_CONF` to `disable` in your Kubernetes configuration.
```
Comment (mirii1994) [inline _source/logzio_collections/_log-sources/iks.md:93]:
```suggestion
```
No need for this in the default part.

File: _source/logzio_collections/_log-sources/iks.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+-n kube-system
+```
+
+##### Configure Fluentd
+
+Download one of these three depending on which one fits your requirements:
+
+* [RBAC DaemonSet](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-rbac.yaml)
+* [non-RBAC DaemonSet](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset.yaml)
+* [Containerd Daemonset](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml)
```
Comment (mirii1994) [inline _source/logzio_collections/_log-sources/iks.md:135]:
```suggestion
Download our [Containerd Daemonset](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml).
```

File: _source/logzio_collections/_log-sources/iks.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+kubectl apply -f /path/to/logzio-daemonset-rbac.yaml
+```
+
+For a non-RBAC DaemonSet, run:
+
+```shell
+kubectl apply -f /path/to/logzio-daemonset.yaml
+```
+
+For a container runtime Containerd, run:
```
Comment (mirii1994) [inline _source/logzio_collections/_log-sources/iks.md:169]:
```suggestion
Run:
```

File: _source/logzio_collections/_log-sources/iks.md (hunk 3 above)
Comment (mirii1994) [reply _source/logzio_collections/_log-sources/iks.md:114]: This one need to be changed too :)

File: _source/logzio_collections/_log-sources/iks.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+#####  Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours,
+and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs,
+see [log shipping troubleshooting](https://docs.logz.io/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+
+## Disabling systemd input
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/iks.md:147]:
```suggestion
### Disabling systemd input
```

File: _source/logzio_collections/_log-sources/iks.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+```shell
+kubectl create secret generic logzio-logs-secret \
+--from-literal=logzio-log-shipping-token='<<SHIPPING-TOKEN>>' \
+--from-literal=logzio-log-listener='https://<<LISTENER-HOST>>:8071' \
+-n kube-system
+```
+
+##### Configure Fluentd
+
+Download our [Containerd Daemonset](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml).
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/iks.md:110]:
```suggestion
Download Logz.io's [Containerd Daemonset](https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml).
```

**Pull request merged**